trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require '.\decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
```

Comment:
- [ ] I think you should use if condition here since the word should only be trimmed if only the word is longer than 10 characters as stated in the project requirement. :blush:

Reply:
hi @Owusu-Desmond, I hope you would be fine. I think it would be nice if you just spare a minute to clone my project and run this code. Then you would be able to know that this code is only trimming if the word is longer than 10 characters. Please kindly see the screenshots below:
- When a word is less than 10 characters, it's not trimming
![shot1](https://github.com/zunairkhan811/OOP_School_Library/assets/116083031/1261e3fd-d955-41fe-ad98-c60699e362d6)
- When a word is more than 10 characters, it's trimming
![shot2](https://github.com/zunairkhan811/OOP_School_Library/assets/116083031/dfe81456-6910-4ac4-aecd-f5f5c2a2f764)

Reply:
Hi @zunairkhan811, Yes it will but per the project requirement if condition should be used.

Reply:
@zunairkhan811 , This is Right. @Owusu-Desmond , the code achieves the same result